source/includes/steps-fle-create-data-encryption-key.yaml

```diff
@@ -183,7 +182,7 @@ content: |
         :tabid: python
 
         .. code-block:: python
-           :emphasize-lines: 13,14,20,30
+           :emphasize-lines: 13,21
```

Comment:
The only thing I'm not sure about is these line number highlights. Is there any way to render them before we merge?

Reply:
sure, just cd into the root level and run 'make html'. This will build the source into html. The resulted build is located in `{root}/build/{your_branch_name}/html'

Reply:
Thanks, also need to use Python 2.7 and install giza and then it worked seamlessly. I added a few more minor fixes.